content/chat/getting-started/javascript.textile

```diff
@@ -0,0 +1,195 @@
+
+* Add the following lines to your @getStarted()@ function to create an instance of a room, attach to the room instance, and then register a listener to subscribe to messages sent to the room. You then also send your first message. Afterwards, run it:
+
+```[javascript]
+const room = await chatClient.rooms.get('my-first-room', AllFeaturesEnabled);
+
+await room.attach();
+
+const {unsubscribe: messageUnsubscribe} = room.messages.subscribe((message: MessageEvent) => {
+    console.log(`Received message: ${message.message.text}`);
+});
+
+const myFirstMessage = await room.messages.send({text: 'My first message!'});
+```
+
+* Use the Ably CLI to send a message to your first room: @ably rooms messages send my-first-channel 'Hello!'@. The message will be received by the client you've subscribed to the room, and be logged to the console.
+
+h2(#step-3). Step 3: Edit a message
+
+If your client makes a typo, or needs to update their original message then they can edit it.
+
+* Add the following line to your @getStarted()@ function to edit your original first message. Technically this will be your second message that you edit, as running the function again will re-send the original message.
+
+```[javascript]
+await room.messages.update(myFirstMessage.copy( {text: 'My 1st message! (edited)'}));
+```
+
+
+h2(#step-4). Step 4: Show who is typing a message
+
+Typing indicators enable you to display messages to clients when someone is currently typing. An event is emitted when someone starts typing, when they press a keystroke, and then another event is emitted after a configurable amount of time has passed without a key press.
+
+* In practice the @typing.start()@ method would be called on keypress, however for demonstration purposes we will call it and wait for the default period of time to pass for a stop event to be emitted. Add the following lines to @getStarted@ to subscribe to typing events and emit one, then run it:
+
+```[javascript]
+const {unsubscribe: typingUnsubscribe} = room.typing.subscribe((typingEvent) => {
+  {
+    if (typingEvent.currentlyTyping.size === 0) {
+      console.log('No one is currently typing');
+    } else {
+      console.log(`${[...typingEvent.currentlyTyping][0]} is typing`);
+    }
+  }
+});
+
+await room.typing.start()
+```
+
+* Use the Ably CLI to subscribe to typing events: @ably rooms typing subscribe my-first-room@. You will see that the client in your IDE starts to type after re-running @getStarted()@, with the message @my-first-client is typing...@.
+
+h2(#step-5). Step 5: Display online status
+
+Display the online status of clients using the presence feature. This enables clients to be aware of one another if they are present in the same room. This enables you to show clients who else is online, provide a custom status update for each, and notify the room when someone enters it, or leaves it, such as by going offline.
```

Comment:
This section starts two consecutive sentence with "this enables" - perhaps worth finding an alternative for one case to make it a nicer read?